docker-compose.yml

```
@@ -0,0 +1,36 @@
+version: '3'
+
+services:
+
+  db:
+    image: postgres:14
+    command: postgres -c 'max_connections=250'
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=changeme
+    volumes:
+      - "dbdata:/var/lib/postgres/data"
+    ports:
+      - "5432:5432"
+
+  redis:
+    image: redis:latest
+    ports:
+      - "6379:6379"
+
+  web:
+    platform: linux/x86_64
+    build: .
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+      - redis
+    environment:
+      - DATABASE_URL=postgresql://db/lettuceMeetApp_development
+      - REDIS_URL=redis://redis:6397
```

if you set the postgres username/password here, apply it here, too
```suggestion
      - REDIS_URL=redis://redis:6397
      - DB_USER=postgres
      - DB_PASS=changeme
```
      - DB_PASS=changeme